docker/config.sh

```diff
@@ -2,7 +2,7 @@
 
 # Logic in this file is used within builds of docker containers
 
-COMMON_APT_PACKAGES="curl zip unzip git wget ca-certificates lsb-release build-essential sudo postgresql-client-15 libpq-dev jq locales"
+COMMON_APT_PACKAGES="curl zip unzip git wget ca-certificates lsb-release build-essential sudo postgresql-client-15 libpq-dev jq locales libkrb5-dev"
```

Comment:
Needed to install arcgis python package. Even if we don't go that route, might leave, it's a pretty insignificant package to include